Libraries/test/TestWebApp/LambdaFunction.cs

```diff
@@ -5,16 +5,15 @@
 
 namespace TestWebApp
 {
-    public class LambdaFunction : APIGatewayProxyFunction
+    public class LambdaFunction : APIGatewayProxyFunction<Startup>
     {
         public const string BinaryContentType = "application/octet-stream";
 
         protected override void Init(IWebHostBuilder builder)
         {
             builder
                 .UseApiGateway()
```

Comment:
Both of these remaining `Use` methods are already called in `CreateWebHostBuilder`, but it would look odd to have `Init` simply return the builder, I thought, so I left it alone. Unless `Init` became virtual instead of abstract? Way out of scope, I understand.

Reply:
The more I think about it, the more it makes sense. An `IWebHostBuilder` created by the typed variant doesn't _require_ any other setup. I think it should be overridden to provide a default implementation of `Init`.